packages/cli/src/endpoints/credentials.endpoints.ts

```diff
@@ -0,0 +1,385 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable import/no-cycle */
```

Comment:
All are needed, unless you mean turning them into line exceptions. For `Server.ts` we have not been doing this, possibly because so many line exceptions would reduce readability. Let me know.